lib/mayaUsd/ufe/UsdContextOps.cpp

```diff
@@ -32,6 +32,35 @@
 
 namespace {
 
+// Ufe::ContextItem strings
+// - the "Item" describe the operation to be performed.
+// - the "Label" is used in the context menu (can be localized).
+static constexpr char kUSDLayerEditorItem[] = "USD Layer Editor";
```

Comment:
Nit-pick: is this some MSVC-specific requirement?  Because otherwise, you're in an unnamed namespace, you don't need the static linkage specifier.